### PR TITLE
fix nav background color on mobile (MLNN-1095)

### PR DESCRIPTION
**Pull request by joshreini1**
Before:
Screen Shot 2023-11-08 at 8 14 57 AM

After:
Screen Shot 2023-11-08 at 9 54 47 AM

### CODE REVIEW

File: docs/stylesheets/style.css [hunk 1]
```diff
@@ -632,11 +632,13 @@ p {
 @media only screen and (max-width: 768px) {
   .nav__items {
     flex-direction: column;
+    display: flex;
```
Comment (walnutdust) [inline docs/stylesheets/style.css:635]: this is probably not needed - will be inherited from line 617 if not overwritten by default!

File: docs/stylesheets/style.css [hunk 2]
```diff
@@ -632,11 +632,13 @@ p {
...
     position: fixed;
-    top: calc(calc(var(--gap-lg) * 2 + 32px) + 0.5rem);
+    top: 0;
```
Comment (walnutdust) [inline docs/stylesheets/style.css:637]: @joshreini1 this variable should be controlling where the first nav item starts to allow our logo and the close icon to be shown - I think we might want to keep it, but wanted to confirm with you

File: docs/stylesheets/style.css [hunk 3]
```diff
@@ -632,11 +632,13 @@ p {
...
     left: 0;
-    width: 100vw;
-    background-color: var(--color-navigation);
+    width: 100%;
```
Comment (walnutdust) [inline docs/stylesheets/style.css:639]: in this context there isn't a difference in 100vw (100 % of viewport width) vs 100%, so it's okay to keep it as before

File: docs/stylesheets/style.css [hunk 4]
```diff
@@ -632,11 +632,13 @@ p {
...
+    height: min-content;
+    background-color: var(--color-navigation) !important;
```
Comment (walnutdust) [inline docs/stylesheets/style.css:641]: generally avoid !important in CSS if possible, this causes headaches when trying to debug and figuring out the precedence of the lines

**Review by walnutdust — CHANGES_REQUESTED** [review]
@joshreini1  thanks for taking a stab at this - it looks like the only change that needs to be done is adding `height: min-content` to achieve our intended goal, could we see if the result of that change would meet our needs?

File: docs/stylesheets/style.css (hunk 3 above)
Comment (joshreini1) [reply docs/stylesheets/style.css:639]: With the viewport still there, I'm seeing this width in my mobile simulator (https://www.webmobilefirst.com/en/). 
Screen Shot 2023-11-08 at 11 38 49 AM

File: docs/stylesheets/style.css (hunk 4 above)
Comment (joshreini1) [reply docs/stylesheets/style.css]: Tested and removed - thanks!

File: docs/stylesheets/style.css (hunk 2 above)
Comment (joshreini1) [reply docs/stylesheets/style.css]: Thanks! Tested and removed this change.

File: docs/stylesheets/style.css (hunk 1 above)
Comment (joshreini1) [reply docs/stylesheets/style.css]: Thanks! Tested and removed this change

File: docs/stylesheets/style.css (hunk 3 above)
Comment (walnutdust) [reply docs/stylesheets/style.css:638]: oh interesting - then yes let's keep it

**Review by walnutdust — APPROVED** [review]

**Pull request merged**